firebase-firestore/CHANGELOG.md

```diff
@@ -1,4 +1,11 @@
-# Unreleased (21.3.1)
+# Unreleased (21.3.2)
+- [changed] Transactions previously required that every document read in a
+  transaction must also be written to. This requirement has been removed, such
```

Comment:
Avoid complicated or legalistic phrasing like "such that". It's better to use plain language. Consider:

> This required has been removed and you can now ...